src/providers/ipa/ipa_subdomains.h

```diff
@@ -69,8 +69,15 @@ errno_t ipa_subdomains_init(TALLOC_CTX *mem_ctx,
                             struct dp_method *dp_methods);
 
 /* The following are used in server mode only */
+enum ipa_trust_type {
+    IPA_TRUST_UNKNOWN = 0,
```

Comment:
In reality IPA_TRUST_UNKNOWN should never be set, because `ipa_server_get_trust_type()` and `ipa_server_decode_trust_type()` fallback to the `IPA_TRUST_AD` type when Trust information stored in IPA LDAP does not contain `ipaPartnerTrustType`, meaning IPA is older version than what supports IPA IPA Trust.

But I add IPA_TRUST_UNKNOWN here as best practice with `enum` declaration.